doc/users/annotations.rst

```diff
@@ -51,9 +51,7 @@ coordinates, one could do::
                 )
 
 For physical coordinate systems (points or pixels) the origin is the
-(bottom, left) of the figure or axes.  If the value is negative,
-however, the origin is from the (right, top) of the figure or axes,
-analogous to negative indexing of sequences.
+(bottom, left) of the figure or axes.
```

Comment:
'bottom, left' sounds more natural to me and matches the (row, col) numpy indexing.  

The inconsistency was with the now deleted 'right, top'.